### PR TITLE
Reset mistakenly overwritten config & output report when running workloads locally in Docker container

### PR DESCRIPTION
**Pull request by ionicsolutions**
Parts of this file were mistakenly deleted when PR #266 was squash-merged. This PR resets it to the state of #254

### CODE REVIEW

File: config/configs_basic.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ timeseries_analyzer:
   auto_detection: True
   id_col: 'ifa'
   tz_offset: 'local'
-  inspection: False
+  inspection: True
```
Comment (VishnuGowthem) [inline config/configs_basic.yaml:69]: inspection can remain "False" in configs_basic.yaml
in PR #254 also it is False.
in configs_full.yaml it is set to True

**Review by VishnuGowthem — COMMENTED** [review]
Other changes are good to proceed

File: config/configs_basic.yaml (hunk 1 above)
Comment (ionicsolutions) [inline config/configs_basic.yaml:69]:
```suggestion
  inspection: False
```

**Comment by ionicsolutions** [comment]
@VishnuGowthem Setting `inspection` to `False` causes an error in both the `demo` and the `local` workflows:

```shell
FileNotFoundError: [Errno 2] No such file or directory: '/output/report_stats/stats_dt_1_1.csv'
```

It seems that the report generation is searching for some output that is created when `inspection` is `True`?

**Comment by VishnuGowthem** [comment]
> @VishnuGowthem Setting `inspection` to `False` causes an error in both the `demo` and the `local` workflows:
> 
> ```shell
> FileNotFoundError: [Errno 2] No such file or directory: '/output/report_stats/stats_dt_1_1.csv'
> ```
> 
> It seems that the report generation is searching for some output that is created when `inspection` is `True`?

@ionicsolutions  Yes seems like the case, I will look into this today.

**Comment by VishnuGowthem** [comment]
> > @VishnuGowthem Setting `inspection` to `False` causes an error in both the `demo` and the `local` workflows:
> > ```shell
> > FileNotFoundError: [Errno 2] No such file or directory: '/output/report_stats/stats_dt_1_1.csv'
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > It seems that the report generation is searching for some output that is created when `inspection` is `True`?
> 
> @ionicsolutions Yes seems like the case, I will look into this today.

Basic report was not set to True in configs_basic.yaml and I verified with True in local successfully, hence made the same change here. 

Seems when Inspection is set to False, 
the full demo run fails (will check further and raise an issue on this if required to investigate it further) 
and Basic Report runs fine. 

And when Inspection is set to True, 
both runs passes,
but for Basic report as inspection step is not a need necessity and it would run slightly longer so we can keep it for false for now. Thanks !

**Comment by ionicsolutions** [comment]
@VishnuGowthem Thanks for investigating! Let's get this PR merged?

**Pull request merged**